File: src/main/java/com/polytech/webscraipper/sdk/LangfuseSDK.java
package com.polytech.webscraipper.sdk;

import org.springframework.stereotype.Service;

@Service
public class LangfuseSDK {
  public final Prompts prompts;
  public final Traces traces;

  public LangfuseSDK(Prompts.PromptsClient promptsClient, Traces.TracesClient tracesClient) {
    this.prompts = new Prompts(promptsClient);
    this.traces = new Traces(tracesClient);
  }
}

File: src/main/java/com/polytech/webscraipper/sdk/LangfuseSDK.kt
package com.polytech.webscraipper.sdk

import org.springframework.stereotype.Service
@Service
class LangfuseSDK(
    promptsClient: Prompts.PromptsClient,
    tracesClient: Traces.TracesClient,
) {
    @JvmField
    val prompts = Prompts(promptsClient)

    @JvmField
    val traces = Traces(tracesClient)
}
